packages/jest-runtime/src/index.ts

```diff
@@ -474,9 +476,30 @@ class Runtime {
     // CJS loaded via `import` should share cache with other CJS: https://github.com/nodejs/modules/issues/503
     const cjs = this.requireModuleOrMock(from, modulePath);
 
+    const transformedCode = this._fileTransforms.get(modulePath);
+
+    let cjsExports: ReadonlyArray<string> = [];
+
+    if (transformedCode) {
+      const {exports} = parseCjs(transformedCode.code);
```

Comment:
this ignore re-exports - I'm not sure what the semantics are? Where would I get the "value" fomr?